tests/utils/googleAnalytics/set.test.ts

```diff
@@ -0,0 +1,39 @@
+import * as faker from 'faker';
+
+import * as initUtils from '../../../src/utils/googleAnalytics/initialize';
+import {set} from '../../../src/utils/googleAnalytics/set';
+
+describe('googleAnalyticsHelper.set', () => {
+  const setUp = () => {
+    const name = faker.lorem.word();
+    const params = {foo: 'bar'};
```

Comment:
```suggestion
    const params = {foo: faker.lorem.text() };
```
faker.js 활용해봐도 좋을 것 같습니다!!

Reply:
우와 좋습니다.